ldap/servers/slapd/pblock.c

```diff
@@ -368,3865 +365,5944 @@ slapi_pblock_clone(Slapi_PBlock *pb)
 #define SLAPI_PBLOCK_GET_PLUGIN_RELATED_POINTER(pb, element) \
     ((pb)->pb_plugin == NULL ? NULL : (pb)->pb_plugin->element)
 
-int
-slapi_pblock_get(Slapi_PBlock *pblock, int arg, void *value)
+static int32_t
+slapi_pblock_get_backend(Slapi_PBlock *pblock, void *value)
 {
+    Slapi_Backend *be = pblock->pb_backend;
 
-#ifdef PBLOCK_ANALYTICS
-    pblock_analytics_record(pblock, arg);
-#endif
```

Comment:
This analytics code (and the one in `_set`) were removed. Should we still keep it? Or it's not used?

Reply:
> This analytics code (and the one in `_set`) were removed. Should we still keep it? Or it's not used?

I'm not 100% what these are for.  I assumed it was for measuring how we accessed the pblock, but I have a feeling this is no longer needed.  Perhaps we can remove it in a separate PR?